test/index.js

```diff
@@ -14,12 +14,24 @@ test('parse selector', (t) => {
   t.end()
 })
 
-test('parse selector class with attributes', (t) => {
+test('parse selector class with attributes class as string', (t) => {
   const obj = parse(['p.italic', { class: 'bold' }])
   t.equal(obj.attrs.class, 'italic bold')
   t.end()
 })
 
+test('parse selector class with attributes class as array', (t) => {
+  const obj = parse(['p.italic', { class: ['bold'] }])
+  t.deepEqual(obj.attrs.class, ['italic', 'bold'])
```

Comment:
`preact-hyperscript` supports it (it passes it through `classNames`) https://github.com/queckezz/preact-hyperscript/blob/master/src/index.js#L30